doc/source/management/data_at_rest_encryption.rst

```diff
@@ -585,7 +585,42 @@ on every other key rotation.
 
 This variable simultaneously turns on the encryption of files used by InnoDB for
 full text search using parallel sorting, building indexes using merge sort, and
-online DDL logs created by InnoDB for online DDL. Encryption is available for file merges used in queries and backend processes.
+online DDL logs created by InnoDB for online DDL.
+
+.. _data-at-rest-encryption.undo-tablespace:
+
+InnoDB Undo Tablespace Encryption
+================================================================================
+
+:Availability: This feature is **Alpha** quality
```

Comment:
I still see alpha here, when according to our encryption spreadsheet, it is GA in 8.0.16?

Reply:
Odd. For some reason, it didn't pull in the latest version. Fixed.